modules/miscutil/lib/upgrades/invenio_2014_05_26_new_index_country.py

```diff
@@ -0,0 +1,224 @@
+            remove_html_markup = 'No',
+            remove_latex_markup = 'No',
+            tokenizer = 'BibIndexCountryTokenizer'
+        """,
+        (index_last_used_id + 1,)
+    )
+
+    run_sql(
+        """
+        CREATE TABLE IF NOT EXISTS idxWORD%02dF (
+          id mediumint(9) unsigned NOT NULL auto_increment,
+          term varchar(50) default NULL,
+          hitlist longblob,
+          PRIMARY KEY  (id),
+          UNIQUE KEY term (term)
+        ) ENGINE=MyISAM;
+        """ % index_id
+    )
+    run_sql(
+        """
+        CREATE TABLE IF NOT EXISTS idxWORD%02dR (
+          id_bibrec mediumint(9) unsigned NOT NULL,
+          termlist longblob,
+          type enum('CURRENT','FUTURE','TEMPORARY') NOT NULL default 'CURRENT',
+          PRIMARY KEY (id_bibrec,type)
+        ) ENGINE=MyISAM;
+        """ % index_id
+    )
+    run_sql(
+        """
+        CREATE TABLE IF NOT EXISTS idxPAIR%02dF (
+          id mediumint(9) unsigned NOT NULL auto_increment,
+          term varchar(100) default NULL,
+          hitlist longblob,
+          PRIMARY KEY  (id),
+          UNIQUE KEY term (term)
+        ) ENGINE=MyISAM;
+        """ % index_id
+    )
+    run_sql(
+        """
+        CREATE TABLE IF NOT EXISTS idxPAIR%02dR (
+          id_bibrec mediumint(9) unsigned NOT NULL,
+          termlist longblob,
+          type enum('CURRENT','FUTURE','TEMPORARY') NOT NULL default 'CURRENT',
+          PRIMARY KEY (id_bibrec,type)
+        ) ENGINE=MyISAM;
+        """ % index_id
+    )
+    run_sql(
+        """
+        CREATE TABLE IF NOT EXISTS idxPHRASE%02dF (
+          id mediumint(9) unsigned NOT NULL auto_increment,
+          term text default NULL,
+          hitlist longblob,
+          PRIMARY KEY  (id),
+          KEY term (term(50))
+        ) ENGINE=MyISAM;
+        """ % index_id
+    )
+    run_sql(
+        """
+        CREATE TABLE IF NOT EXISTS idxPHRASE%02dR (
+          id_bibrec mediumint(9) unsigned NOT NULL,
+          termlist longblob,
+          type enum('CURRENT','FUTURE','TEMPORARY') NOT NULL default 'CURRENT',
+          PRIMARY KEY (id_bibrec,type)
+        ) ENGINE=MyISAM;
+        """ % index_id
+    )
+
+    # Field and index to field mapping
+    field_id = run_sql(
+        """
+        INSERT INTO field
+        SET
+            name = 'country',
+            code = 'country'
+        """
+    )
+
+    run_sql(
+        """
+        INSERT INTO idxINDEX_field (id_idxINDEX, id_field)
+        VALUES (%d,%d)
+        """,
+        (index_id, field_id)
+    )
+
+    # Tags
+    tag_100__u_id = run_sql(
+        """
+        SELECT id FROM tag
+        WHERE value = '100__u'
+        """
+    )
+    tag_700__u_id = run_sql(
+        """
+        SELECT id FROM tag
+        WHERE value = '700__u'
+        """
+    )
+    tag_110__u_id = run_sql(
+        """
+        INSERT INTO tag
+        SET
+            name = 'SPIRES name'
+            value = '110__u'
```

Comment:
All this section is installation-specific (e.g. "SPIRES name").  We usually leave it out in upgrade recipes, distinguishing between `do_upgrade()` and `do_upgrade_atlantis()`. See for example `invenio_2013_10_18_new_index_filetype.py` and other past upgrade recipes.  We should leave it out here too...